spacy/pipeline/entity_linker.py

```diff
@@ -474,18 +474,24 @@ def predict(self, docs: Iterable[Doc]) -> List[str]:
 
                 # Looping through each entity in batch (TODO: rewrite)
                 for j, ent in enumerate(ent_batch):
-                    sent_index = sentences.index(ent.sent)
-                    assert sent_index >= 0
+                    assert hasattr(ent, "sents")
+                    sents = list(ent.sents)
+                    sent_indices = (
+                        sentences.index(sents[0]),
+                        sentences.index(sents[-1]),
+                    )
+                    assert sent_indices[-1] >= sent_indices[0] >= 0
```

Comment:
```suggestion
                    assert 0 >= sent_indices[0] >= sent_indices[1]
```
This should be the same but feels more natural to me, as the code below uses these exact formulations. We know that `sent_indices` is always of length 2 so we might as well just index directly with `0` and `1`.